yoke/assets/joypad/base.js

```diff
@@ -221,11 +282,11 @@ Pedal.prototype._truncate = function(f) {
     return f;
 };
 Pedal.prototype.onAttached = function() {
+    this._handbrake = getComputedStyle(this.element)['animation-name'] == 'handbrake';
```

Comment:
Are we going back to `animation-name` settings?

Reply:
Well, that's embarassing.

If there are any more lines about "handbrake" or `this._handbrake`, they're a mistake. I was experimenting on how to repurpose the pedal controls, but it was getting too complicated for very little benefit, so I crossed that out. I left that line in by accident.

I'm not going back there. Would you please delete that line for me? Or is there a way for me to delete it without creating another pull request?

Reply:
I think the commands to redownload a branch are:

```bash
git checkout future # switch to future branch
git fetch origin
git reset --hard origin/future
```

Be careful - this deletes all changes. I will apply the fix now, though, I want to see this PR merged and don't want you to focus on not-so-important Git things when your strong side is JS :)